modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue

```diff
@@ -252,108 +196,67 @@ export default {
         }
       }
       return instance;
-    },
-    getLabel(id, index) {
-      const titleField = this.field.titleField || null;
-      const item = this.items.find(item => item._id === id);
-      return item.schemaInput.data[titleField] || `Item ${index + 1}`;
-    },
-    openInlineItem(id) {
-      this.items.forEach(item => {
-        item.open = (item._id === id) || this.alwaysExpand;
-      });
-    },
-    closeInlineItem(id) {
-      this.items.forEach(item => {
-        item.open = this.alwaysExpand;
-      });
     }
   }
 };
 
 function modelItems(items, field) {
-  return items.map(item => {
-    const open = alwaysExpand(field);
+  return items.map((item) => {
     return {
       _id: item._id || cuid(),
       schemaInput: {
         data: item
-      },
-      open
+      }
     };
   });
 }
-
-function alwaysExpand(field) {
```

Comment:
Hmm, switching to this tabular approach for all inline arrays does indeed imply that control over whether to "expand" each row is going away. I think that is probably OK, but @stuartromanek might see it differently, in which case we might need to revisit offering inline arrays in different formats.